test/stablex/stablecoin_converter.js

```diff
@@ -467,6 +466,77 @@ contract("StablecoinConverter", async (accounts) => {
       assert.equal((await stablecoinConverter.getBalance.call(user_2, erc20_2)).toString(), basicTrade.deposits[1].amount.sub(getExecutedSellAmount(fullBuyVolumes[1], prices[0], prices[1])).toString(), "Sold tokens were not adjusted correctly")
       assert.equal((await stablecoinConverter.getBalance.call(user_2, feeToken)), fullBuyVolumes[1].toString(), "Bought tokens were not adjusted correctly")
     })
+    it("ensures half of the token imbalance (fees) is burned and that better solutions don't double-burn", async () => {
```

Comment:
I agree with @josojo  that the "and" in the test description is an indicator that it should maybe be 2 tests. Also to make it more exhaustive we could submit two solutions even in the first auction (to make sure we are burning the larger amoun). But no strong feelings.

Reply:
Yes! I like that idea, although please see the following response regarding code duplication.

https://github.com/gnosis/dex-contracts/pull/310#discussion_r349038603

Reply:
I will create a separate issue in correspondence with this "Nice to Have".